src/containers/teleport/components/tabList/TabList.tsx

```diff
@@ -0,0 +1,45 @@
+import { useNavigate } from 'react-router-dom';
+
+import ButtonTeleport from 'src/containers/warp/components/buttonGroup/indexBtn';
+import { TypePages } from '../../type';
+import s from './TabList.module.scss';
+
+type TabListProps = {
+  selected: TypePages;
+};
+
+function TabList({ selected }: TabListProps) {
+  const navigate = useNavigate();
+
+  const handleHistory = (to: string) => {
+    navigate(to);
```

Comment:
when write such code, need to think isn't this a link?
link allow more browser UX

Reply:
need to change